README.md

```diff
@@ -59,6 +59,21 @@ https://www.npmjs.com/package/do_username
 
 `npm install do_username`
 
+## CLI
+
+```bash
+# Call the CLI command with a max length of 30 characters
+$ do_username
+
+# Call the CLI command with a max length of 10 characters
+$ do_username 10 
+```
+
+```text
+GiantBlueGreenScubaDiver
+BlueMorel
+```
+
```

Comment:
Can this be added to ruby/README.md please, instead of here?

Reply:
Sure thing! Sorry - totally forgot I added instructions to the readme. 😅